erin.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: page
+title: Erin's Awesome Page
+---
+I'm a template.
+
+## What I do
+- Technical writing
+- Twitter lurking
+- Bunny petting
```

Comment:
Unfortunately no, but there's a place in town that has an All You Can Pet Bunny Buffet every Saturday! :heart: